src/main/java/uk/gov/hmcts/ccd/data/casedataaccesscontrol/MultipleQueryRequestResource.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.ccd.data.casedataaccesscontrol;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.extern.jackson.Jacksonized;
+
+import java.util.List;
+
+@Builder
+@Getter
+@Jacksonized
+public class MultipleQueryRequestResource {
+
+    List<RoleAssignmentQuery> queryRequests;
```

Comment:
Should this be `private` access ?

Reply:
Resolved in 1ee8ce11700eb2cfa7b7763bbd00bd9846172417.

The previous use of Lombok's `@Value` generated a linting warning "@Value already marks non-static, package-local fields private".  However as I have switched to use Lombok's `@Getter` I should have re-adjusted the definition.  They now read as both `private` and `final`.